lib/src/actions/file/link.rs

```diff
@@ -210,7 +211,7 @@ mod tests {
             ..Default::default()
         };
 
-        let steps = file_link_action.plan(&manifest, &contexts);
+        let steps = file_link_action.plan(&manifest, &contexts).unwrap();
```

Comment:
You could use a `?` instead of the `.unwrap()` here

Reply:
I'm thinking that since this one is a test, `.unwrap()` is a better option...if it's an error, it'll panic and fail the test anyway.